src/GitVersionCore/VersionCalculation/BaseVersionCalculators/VersionInBranchNameVersionStrategy.cs

```diff
@@ -57,5 +57,12 @@ internal IEnumerable<BaseVersion> GetVersions(string tagPrefixRegex, IBranch cur
 
             return null;
         }
+
+        private static string NameWithoutOrigin(IBranch branch)
```

Comment:
The `NameWithoutOrigin` method can be moved to the property `ReferenceName.WithoutOrigin`, no?

Reply:
Well I was considering that, but it's used only in one place and I was thinking to replace that usage with Name.WithoutRemote as it is more general, but a test was failing and I kept it here as a private method where the usage is.

Reply:
I will leave it there for now